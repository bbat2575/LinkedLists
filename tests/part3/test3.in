NEW 0
NEW 1
{0}
NEW 3
$
{0}
3
NEW 2
{1}
2
NEW 2
1
{2}
NEW 2
{1}
{2}
NEW 3
{0}
{1}
2.2
NEW 3
0
2
{10}
NEW 3
{0}
{100}
fin
