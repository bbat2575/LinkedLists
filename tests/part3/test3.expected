
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: 

> Nested 1: {List 0}

> Nested 2: $ -> {List 0} -> 3

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> 
